### PR TITLE
VUE-34 pagination

### PR DESCRIPTION
**Pull request by emuvente**
Not ready for merging quite yet. I'm just looking to get thoughts & opinions. Also, the loading spinner isn't working yet. There were other ways I could have gone about handling updating the query variables, so if there are any downsides to this I can explore other means.

### CODE REVIEW

**Comment by mcstover** [comment]
Looks like it's coming together well!

File: src/pages/Lend/LendPage.vue [hunk 1]
```diff
@@ -25,55 +32,126 @@
...
+		to({ offset }) {
+			const page = Math.floor(offset / loansPerPage) + 1;
+			return page > 1 ? page : null;
+		},
+		from({ page }) {
+			const pagenum = numeral(page).value() - 1;
+			return { offset: pagenum > 0 ? loansPerPage * pagenum : 0 };
+		}
+	},
+};
```
Comment (BoulderBrains) [inline src/pages/Lend/LendPage.vue]: Would love to walk through this portion of the code with you at some point @emuvente

File: src/pages/Lend/LendPage.vue [hunk 2]
```diff
@@ -25,55 +32,126 @@
...
+.lend-page {
+	background-color: $kiva-bg-lightgray;
+
+	.loan-card-group {
+		position: relative;
+	}
+
+	.loading-overlay {
+		position: absolute;
+		background-color: rgba($kiva-bg-lightgray, 0.7);
```
Comment (BoulderBrains) [inline src/pages/Lend/LendPage.vue]: Is this valid css? I attempted to copy and paste this in the console for a random element on the page and it won't accept this value: background-color: rgba($kiva-bg-lightgray, 0.7);

File: src/components/Kv/KvPagination.vue [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+				</router-link>
+			</li>
+			<li v-else class="pagination-next disabled">
+				<span class="show-for-sr">Next page</span>
+				<kv-icon name="triangle" />
+			</li>
+		</ul>
+	</nav>
+</template>
+
```
Comment (BoulderBrains) [inline src/components/Kv/KvPagination.vue]: Nice logic in here! 👍

File: src/pages/Lend/LendPage.vue (hunk 2 above)
Comment (emuvente) [reply src/pages/Lend/LendPage.vue]: It's not valid css, but it is valid sass. See http://sass-lang.com/documentation/Sass/Script/Functions.html#rgba-instance_method

From the examples:
```
rgba(#102030, 0.5) => rgba(16, 32, 48, 0.5)
rgba(blue, 0.2)    => rgba(0, 0, 255, 0.2)
```

**Comment by BoulderBrains** [comment]
👀👀🆒

**Pull request merged**